src/Selenium2Library/keywords/_element.py

```diff
@@ -272,6 +272,30 @@ def click_element(self, locator):
         self._info("Clicking element '%s'." % locator)
         self._element_find(locator, True, True).click()
 
+    def key_click_element(self, keyname, locator):
```

Comment:
Just to let you know I'm currently working on a test page for this keyword. Should upload an intial attempt in the next few hours...